react-components/src/hooks/network/getAssetsList.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type CogniteClient, type CursorResponse, type Asset } from '@cognite/sdk';
+import { buildFilter } from '../../utilities/buildFilter';
+
+export const getAssetsList = async (
+  sdk: CogniteClient,
+  {
+    query,
+    cursor,
+    limit = 1000,
+    sort = [{ property: ['_score_'] }],
```

Comment:
I dont have a strong opinion on this, but is it possible to put it as a const and a type?